src/test/java/com/salaboy/conferences/agenda/rest/ContractVerifierBase.java

```diff
@@ -1,24 +1,31 @@
 package com.salaboy.conferences.agenda.rest;
 
+import com.salaboy.conferences.agenda.rest.repository.AgendaItemRepository;
 import io.restassured.RestAssured;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
 import org.mockito.MockitoAnnotations;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.boot.web.server.LocalServerPort;
 import org.springframework.test.context.ActiveProfiles;
 
 
-@SpringBootTest( webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
-        properties = "server.port=0")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+        properties = {"server.port=0"})
 @ActiveProfiles("dev")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
It was necessary because we are using `@BeforeAll` annotation and the method setup must be static.

Reply:
perfect.. this makes sense